app/models/bucket_job.rb

```diff
@@ -0,0 +1,9 @@
+class BucketJob < SimpleDelegator
+  attr_reader :summary
+
+  def initialize(job_id, summary)
+    @job = Job.find(job_id.to_i)
```

Comment:
Good start here. Love that we are getting a Ruby object to pass to views and other places 👍 

To make the best use of the Decorator Pattern, you should initialize with the `Job` object as the first parameter instead  of searching for the job. So from the outside it would be:

```
job = Job.first
BucketJob.new(job, "yo")
```

Instead of:
```
job_id = 1
BucketJob.new(job_id, "yo")
```